FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -73,10 +78,9 @@ const GeneralInfo = (props) => {
 
     const fields = {
         'name': {defaultValue: mainProfile?.name},
-        'is_fop': {defaultValue: mainProfile?.is_fop},
         'official_name': {defaultValue: mainProfile?.official_name ?? null},
-        'edrpou': {defaultValue: mainProfile?.edrpou ?? ''},
-        'ipn': {defaultValue: mainProfile?.ipn ?? ''},
+        'edrpou': {defaultValue: mainProfile?.edrpou ?? null},
+        'ipn': {defaultValue: mainProfile?.ipn ?? null},
```

Comment:
Why is null returned if value in mainProfile already could be null or undefined? Should be empty string a optional value?